<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zchxRadarOptWidget</class>
 <widget class="QWidget" name="zchxRadarOptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	height:20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QPushButton" name="btn_1000_minus">
     <property name="text">
      <string>-1000</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_100_minus">
     <property name="text">
      <string>-100</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_10_minus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>-10</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_minus">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_plus">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_10_plus">
     <property name="text">
      <string>+10</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_100_plus">
     <property name="text">
      <string>+100</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_1000_plus">
     <property name="text">
      <string>+1000</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_auto">
     <property name="text">
      <string>自动</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
